README.md

```diff
@@ -0,0 +1,13 @@
+# About Project
+
+This is the twrp.me Jekyll project for the twrp.me website.
+Maintainers submitting new devices are requested to add
+their device markdown page to the correct oem directory.
+This makes sure that we add the correct information to the
+web site when a new devices is created.
+
+# Submitting
+Please use https://gerrit.twrp.me to submit new device
+markdowns.
```

Comment:
Guess you missed this?

Reply:
If you want to add your device please email me with your device tree and tree you used to compile under at bigbiff@teamw.in

Closing this errant PR.